spec/workers/weather_poller_worker_spec.rb

```diff
@@ -1,39 +1,58 @@
 require 'rails_helper'
+
+TESTDATA =
+  {
+    "daily" => {
+      "data" => [
+        {
+          "sunsetTime" => 1_559_573_189,
+          "sunriseTime" => 1_559_573_189
```

Comment:
Even tho it's test data, can we make `sunsetTime > sunriseTime` (or vice-versa if appropriate)?